### ISSUE TITLE
Text-to-speech API: Method failed with status code 403: Forbidden

### ISSUE THREAD

**Issue by amaynut** [watson-developer-cloud/python-sdk#777]
**Overview**
Enable to use IBM Watson API with Python. I keep getting 403 error (forbidden) no matter which URL I used 
I tried the following URLs:
- https://stream.watsonplatform.net/text-to-speech/api
- https://api.us-south.speech-to-text.watson.cloud.ibm.com/v1/synthesize
- https://api.us-south.speech-to-text.watson.cloud.ibm.com/text-to-speech/api
- https://api.us-south.speech-to-text.watson.cloud.ibm.com

But none of them work.

#### Code sample
```python
import json
from os.path import join, dirname
from ibm_watson import TextToSpeechV1
from ibm_watson.websocket import SynthesizeCallback
from ibm_cloud_sdk_core.authenticators import IAMAuthenticator
from ibm_watson import ApiException

apikey = 'XXXXXXXXXXXXXX'
url = 'https://stream.watsonplatform.net/text-to-speech/api'
try:
    authenticator = IAMAuthenticator(apikey)
    service = TextToSpeechV1(authenticator=authenticator)
    service.set_service_url(url)

    voices = service.list_voices().get_result()
    print(json.dumps(voices, indent=2))
except ApiException as ex:
    print("Method failed with status code " + str(ex.code) + ": " + ex.message)


```

I'm able to use the API via CURL. 

**Expected behavior**
Getting a success response from the API

**Actual behavior**
Getting 403 (forbidden) 

**Screenshots**
![image](https://user-images.githubusercontent.com/5345128/106367843-9c230000-6313-11eb-8ec7-b93ade72b5b1.png)

**SDK Version**
Latest

**Additional information:**
- OS: Windows 10
- Python 3.9.1

**Comment by chughts** [comment]
See the answer to your question on stack overflow - https://stackoverflow.com/questions/65973220/ibm-watson-text-to-speech-api-method-failed-with-status-code-403-forbidden/65990867#65990867

the url should be `https://api.us-east.text-to-speech.watson.cloud.ibm.com`, if your instance is US east.

**Comment by SirSpidey** [comment]
> I'm able to use the API via CURL.

I don't think that the endpoint is the issue if Curl succeeds. The older endpoints still work for now (although deprecated), so `stream.watsonplatform.net/text-to-speech/api` and `api.us-south.text-to-speech.watson.cloud.ibm.com` work for the Dallas location with the same API key.

**Comment by mediumTaj** [comment]
The endpoints you list above are for speech-to-text, text to speech and combinations of both. Can you confirm that you are using the text to speech endpoint that you are using in the curl command in your application? It might be helpful to provide both the curl command and sample authentication code you are using in your application (API keys redacted).

**Comment by apaparazzi0329** [comment]
Looking into this issue again, we have located the source of the error and determined a viable solution. We expect a patch release for this in the coming days

**Comment by apaparazzi0329** [comment]
This issue has been patched in version 5.2.1. Closing as resolved
